postgresql-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+org = "ballerinax"
+name = "postgresql"
+version = "@toml.version@"
+
+[[platform.java11.dependency]]
+path = "../postgresql-native/build/libs/postgresql-native-0.1.0.jar"
+groupId = "org.ballerinalang"
+artifactId = "postgresql-native"
+version = "@project.version@"
+
+[[platform.java11.dependency]]
+path = "@project.path@/repo/bala/ballerina/sql/@stdlib.sql.version@/java11/platform/java11/sql-native-@stdlib.sql.native.version@.jar"
```

Comment:
you can copy the sql-native dependency to lib folder like in here. https://github.com/ballerina-platform/module-ballerina-grpc/blob/2f2874f07b7e326ec8650ba379de57414a7efbf8/grpc-ballerina/Ballerina.toml#L22 . if you doing that, you need to change the build.gradle like in here https://github.com/ballerina-platform/module-ballerina-grpc/blob/2f2874f07b7e326ec8650ba379de57414a7efbf8/grpc-ballerina/build.gradle#L51